projects/core/src/radio/radio-group.element.spec.ts

```diff
@@ -73,9 +74,58 @@ describe('cds-radio-group', () => {
 
     component = element.querySelector<CdsRadioGroup>('cds-radio-group');
 
+    await componentIsStable(component);
     const radio1 = component.querySelectorAll('cds-radio')[0];
     const radio2 = component.querySelectorAll('cds-radio')[1];
     expect(radio1.inputControl.name).toBe(radio2.inputControl.name);
     expect(radio1.inputControl.name).toEqual('my-radio');
   });
+
+  // describe('should sync radio-panel controls within a group', async () => {
+  //   element = await createTestElement(html`
```

Comment:
For some reason the test is blowing up here on `createTestElement`, currently investigating

Reply:
I would recommend removing them until we can get it working. If we want to track this, I would recommend opening an issue that details this situation instead of leaving the comments in the source code.